src/index.html

```diff
@@ -11,6 +11,18 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input data-qa="keypress" class="search-form__input search-form__input_big"
+          type="text" placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
+
+    <form class="search-form search-form_small" data-qa="small">
+      <label class="search-form__label search-form__label_small">
+        <input data-qa="hover" class="search-form__input search-form__input_small"
+          type="text" placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
```

Comment:
The same as above, also no need to use tag form here, you can use only one tag form for this task